css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,18 @@
+button {
+  font-size: 24px;
+  padding: 10px 30px;
+}
+
+button:hover {
+  background-color: black;
+  color: white;
+}
```

Comment:
The mouse cursor should also change when hovering over the button:

---

![image](https://user-images.githubusercontent.com/16143771/158653334-631e6a59-433f-482d-b496-f6945aa3cb7e.png)

Reply:
Revised!